### PR TITLE
Fixed issue where websocket hook did not call original completion

### PR DESCRIPTION
**Pull request by ogres**
Send message hook was changing original completion instead of wrapping it in a hook.
Similar hook already exists for receivemessage functionality, but it was not working for send-message, original completion was never called.
Adjusted code to create a new completionHook which records the information and then original completion block.

I also updated API-Key for piesocket and added a message after sending message, to make sure that completion is being called and at least success/failure message is logged.

### CODE REVIEW

**Comment by ogres** [comment]
@NSExceptional could you have a look at this one?

File: Example/FLEXample/MiscNetworkRequests.m [hunk 1]
```diff
@@ -153,7 +153,7 @@ - (void)sendExampleNetworkRequests:(NSURLSession *)sessionWithDelegate {
 }
 
 - (void)sendExampleWebsocketTraffic:(NSURLSession *)sessionWithDelegate {
-    NSString *APIKey = @"oCdCMcMPQpbvNjUIzqtvF1d2X2okWpDQj4AwARJuAgtjhzKxVEjQU6IdCjwm";
+    NSString *APIKey = @"VCXCEuvhGcBDP7XhiJJUDvR1e1D3eiVjgZ9VRiaV";
```
Comment (NSExceptional) [inline Example/FLEXample/MiscNetworkRequests.m:156]: Why did you change the API key? Did the old one expire or something?

File: Classes/Network/PonyDebugger/FLEXNetworkObserver.m [hunk 2]
```diff
@@ -1591,15 +1591,17 @@ typedef void (^SendMessageBlock)(
...
             websocketTask:slf sendMessagage:message
         ];
-        completion = ^(NSError *error) {
+        
+        id completionHook = ^(NSError *error) {
             [FLEXNetworkObserver.sharedObserver
                 websocketTaskMessageSendCompletion:message
                 error:error
             ];
+            completion(error);
```
Comment (NSExceptional) [inline Classes/Network/PonyDebugger/FLEXNetworkObserver.m:1600]: `completion` is nullable, this needs to check for that before calling it or it will crash

**Review by NSExceptional — CHANGES_REQUESTED** [review]
Nice catch, thank you! Also, sorry, I must have totally missed this PR. Thanks for pinging me.

File: Classes/Network/PonyDebugger/FLEXNetworkObserver.m (hunk 2 above)
Comment (NSExceptional) [inline Classes/Network/PonyDebugger/FLEXNetworkObserver.m:1600]:
```suggestion
            if (completion) {
                completion(error);
            }
```

File: Example/FLEXample/MiscNetworkRequests.m (hunk 1 above)
Comment (ogres) [reply Example/FLEXample/MiscNetworkRequests.m:156]: Yes, it returns error `{"error":"Invalid API key"}`.
Based on piesocket's website, its changed every week:

image

**Comment by ogres** [comment]
> Nice catch, thank you! Also, sorry, I must have totally missed this PR. Thanks for pinging me.

I've added nullability check
Thanks for the review 👍
